lib/version_hook.rb

```diff
@@ -1,3 +1,3 @@
 module SqliteVersionHook
-  VERSION = '2.2.1'
```

Comment:
I'd recommend to do those kind of commits and tags directly from master branch. It is not a big deal, just for future releases.

Reply:
It's true, I did no realize.